pom.xml

```diff
@@ -19,20 +19,36 @@
     <build>
         <directory>test-outputs/target</directory>
     </build>
+
     <dependencies>
         <!-- https://mvnrepository.com/artifact/org.seleniumhq.selenium/selenium-java -->
         <dependency>
             <groupId>org.seleniumhq.selenium</groupId>
             <artifactId>selenium-java</artifactId>
-            <version>${seleniumVersion}</version>
+            <version>4.18.1</version>
         </dependency>
 
+
         <!-- https://mvnrepository.com/artifact/org.testng/testng -->
         <dependency>
             <groupId>org.testng</groupId>
             <artifactId>testng</artifactId>
-            <version>${testNGVersion}</version>
+            <version>7.9.0</version>
             <scope>test</scope>
         </dependency>
+
+        <!-- https://mvnrepository.com/artifact/io.github.bonigarcia/webdrivermanager -->
+        <dependency>
+            <groupId>io.github.bonigarcia</groupId>
+            <artifactId>webdrivermanager</artifactId>
+            <version>5.7.0</version>
```

Comment:
you did not need a web driver manager.

Reply:
I use it in the BaseTest Class as shown in the screenshot
![Screenshot 2024-03-17 132639](https://github.com/Ashraaf7/AutomationExercisePracticing/assets/48325308/c3bbc2a4-6239-418e-b523-f70a217610cc)

Reply:
Check here:https://bonigarcia.dev/webdrivermanager/#webdrivermanager-and-selenium-manager
![image](https://github.com/Ashraaf7/AutomationExercisePracticing/assets/69063398/2b92e9ea-3903-441b-854f-993b7cc24437)
@bego00